### DISCUSSION TITLE
Good settings for DoRA training?

### DISCUSSION THREAD

**Discussion by xixxix-GH** [67372a/LoRA_Easy_Training_Scripts#29] [Q&A] [answered]
I've trained few models on sdxl 1.0 and Illustrious 0.1 bases and I'm getting really bad results. I was wondering if someone could share working config.

**Comment by 67372a** [comment]
Hello @Crousere , could you provide your current toml(s) that aren't working well? That would help me baseline and then share an effective config.

**Reply by xixxix-GH** [reply]
Don't have them anymore. Is there no simple baseline for optimizer, scheduler, args, dimensions etc?

**Reply by 67372a** [reply, marked as answer]
@Crousere There are, just that there can be many considerations as to why it isn't working well for you, and knowing your configurations helps. Training is a complex interaction of many factors, data/captions, hyperparameters, optimizer, and a degree of RNG.

Give this a try 
[noob_vp_v1_0_scornm_ssc_v0_29.toml.txt](https://github.com/user-attachments/files/19985974/noob_vp_v1_0_scornm_ssc_v0_29.toml.txt)

**Reply by xixxix-GH** [reply]
I could not make it work, my 16gb 4070 ti super choked to those settings, I adjusted batch size but still got a blurry mess. Now that I have some free time I wonder if DoRA training would work with simple Prodigy setup that trains UNET only for Illustrious 0.1 without EDM2 loss weighting.

I wanted to ask few question regarding that .toml setup you provided. Doesn't Lycoris generally work better with Network Dimension being 2x the Network Alpha? I remember reading that it was originally intended that way. Also why use LR scheduler with restarts and use Number of cycles = 1?

**Reply by 67372a** [reply]
@Crousere Could you explain more by blurry mess? Can you share some examples?

Alpha 2x dim is supported by research generally for lora/locon, academic and personal experience, it increases the effective rank of the network. Simply raising LR isn't comparable. See https://arxiv.org/abs/2410.21228 and https://arxiv.org/abs/2405.09673.

Because we want to use REX, but don't want to do restarts. 1 cycle just means without restarts. Restarts are typically ineffective at small scales, and research seems to also point to them generally not making sense.

**Reply by xixxix-GH** [reply]
Thanks for reply, I have been so busy irl that I haven't had time to test more settings. I wish people who test and find more good settings for DoRA would post their configs here so more people could try them because it looks so promising on paper as lora replacement.
